app/mailers/jobapps_mailer.rb

```diff
@@ -42,6 +42,16 @@ def interview_reschedule(interview)
          reply_to: reply_to
   end
 
+  def send_note_for_later(application_record)
```

Comment:
Not for now, want MVP. Maybe later. Good thought. Don himself has expressed no interest.